LayoutTests/imported/w3c/web-platform-tests/html/canvas/element/text/2d.text.measure.actualBoundingBox.html

```diff
@@ -56,6 +56,9 @@ <h1>2d.text.measure.actualBoundingBox</h1>
   _assert(ctx.measureText('ABCD').actualBoundingBoxAscent >= 85, "ctx.measureText('ABCD').actualBoundingBoxAscent >= 85");
   _assert(ctx.measureText('ABCD').actualBoundingBoxDescent >= 37, "ctx.measureText('ABCD').actualBoundingBoxDescent >= 37");
 
+  _assert(Number.isInteger(ctx.measureText('D').actualBoundingBoxRight) == false, "Number.isInteger(ctx.measureText('D').actualBoundingBoxRight) == false");
+  _assert(Number.isInteger(ctx.measureText('ABCD').actualBoundingBoxRight) == false, "Number.isInteger(ctx.measureText('ABCD').actualBoundingBoxRight) == false");
+
```

Comment:
I am sorry. I think I confused you by my question about `glyphOverflow` in `widthForSimpleText()` and `widthForComplexText()`. I just realized it is tested in the line above:
```
_assert(ctx.measureText('D').actualBoundingBoxRight >= 75, "ctx.measureText('D').actualBoundingBoxRight >= 75");
```
`CanvasRenderingContext2DBase::measureTextInternal()` uses `glyphOverflow` in calculating `actualBoundingBoxLeft` and `actualBoundingBoxRight`.
```
    metrics->setActualBoundingBoxLeft(glyphOverflow.left - offset.x());
    metrics->setActualBoundingBoxRight(fontWidth + glyphOverflow.right + offset.x());
```

Reply:
So the new assertions you added are meaningless. The TextMetrics.idl says they are `double`. Please remove them.